### DISCUSSION TITLE
How do you treat Longitudinally polarized massive vector bosons (W, Z bosons) separately in the FeynCalc package?

### DISCUSSION THREAD

**Discussion by jaydesai10** [FeynCalc/feyncalc#156] [Q&A]
I am trying to compute Standard Model cross-sections for Electroweak processes. For the interactions I am looking at, I have initial state W+ and W- vector bosons. How do you specify that the initial state vector bosons be only longitudinally polarized. For example, to only include transverse polarization, you set the option of the function FCFAConvert: TransversePolarizationVectors -> {}

Is there a similar option to only consider longitudinally polarized W bosons in my initial state?

Edit: (Additional Question) Also, when I am performing PolarizationSum using the DoPolarizationSums method, is there a way to only include the longitudinal polarization here and ignore the transverse polarization contribution?

**Comment by vsht** [comment]
> For example, to only include transverse polarization, you set the option of the function FCFAConvert: TransversePolarizationVectors -> {}

This option merely states that `ep(k).k = 0` which is completely unrelated to the type of polarizations involved.
For example, the generic polarization vector of a massive particle equally satisfies `ep(k).k = 0` but also comprises longitudinal polarizations.

> Is there a similar option to only consider longitudinally polarized W bosons in my initial state?

> Edit: (Additional Question) Also, when I am performing PolarizationSum using the DoPolarizationSums method, is there a way to only include the longitudinal polarization here and ignore the transverse polarization contribution?

As of now there are no built-in routines to pick out specific polarizations. But especially for longitudinal polarizations
it is quite easy to replace ep(k) with the standard formula `ep^mu (k) = k^mu/M -M/n.k k^mu` using a suitably chosen auxiliary
vector. Or take one of the existing choices from hep-ph/0011075.

**Reply by jaydesai10** [reply]
> This option merely states that ep(k).k = 0 which is completely unrelated to the type of polarizations involved.
For example, the generic polarization vector of a massive particle equally satisfies ep(k).k = 0 but also comprises longitudinal polarizations.

This has lead me to a very naive question. The condition `ep(k).k = 0` should hold true generally for on-shell massive gauge bosons. But, when I try to compute the amplitude of a process where the initial states are W bosons, I have terms of type `ep(k).k`  . Shouldn't those vanish even at the amplitude level?

**Reply by jaydesai10** [reply]
Also, I am new using FeynCalc and Mathematica. How do I replace ep(k) with something inside the expression of the amplitude that I have from FeynCalc?

**Reply by vsht** [reply]
> This has lead me to a very naive question. The condition `ep(k).k = 0` should hold true generally for on-shell massive gauge bosons. But, when I try to compute the amplitude of a process where the initial states are W bosons, I have terms of type `ep(k).k` . Shouldn't those vanish even at the amplitude level?

If you set the `Transversality` or `TransversePolarizationVectors` options they will, otherwise not. 

> Also, I am new using FeynCalc and Mathematica. How do I replace ep(k) with something inside the expression of the amplitude that I have from FeynCalc?

With something like `exp /. Momentum[Polarization[p, _], ___] :> Momentum[...]` 

But if you are new to FeynCalc
you should first go through the tutorials:
https://github.com/FeynCalc/feynknowledge/tree/master/Tutorials

And if you are new to Mathematica, then you should probably start with understanding how Mathematica works,
since otherwise it is really difficult to do sth sensible with FeynCalc.

**Reply by jaydesai10** [reply]
Thank you!
